include/variables.h

```diff
@@ -3082,6 +3082,7 @@ extern f64 D_801E0EB0;
 // extern UNK_TYPE4 D_801E1068;
 extern UNK_PTR D_801E10B0;
 extern const s16 gAudioTatumInit[];
+#define gTatumsPerBeat (gAudioTatumInit[1])
```

Comment:
What?
Could you explain what's the purpose of this define?

Reply:
This was a remnant of OoT and was mainly used before `gAudioTatumInit` was named. `gAudioTatumInit[0]` still represents something unknown so I'm still not sure if `gAudioTatumInit` is a good name or not. Nonetheless, this really shouldn't be in variables.h, so I've moved it into audio_heap.c